src/request_items.rs

```diff
@@ -210,10 +214,17 @@ impl RequestItems {
                 RequestItem::JsonField(key, value) => {
                     body.insert(key, value);
                 }
+                RequestItem::JsonFieldFromFile(key, value)
+                | RequestItem::DataFieldFromFile(key, value) => {
+                    let mut file = File::open(value)?;
+                    let mut contents = String::new();
+                    file.read_to_string(&mut contents)?;
+                    body.insert(key, serde_json::from_str(&contents)?);
```

Comment:
I think this is wrong, if it's a `DataFieldFromFile` then it should be inserted as a string, not decoded as JSON